network-mapper/templates/mapper-servicemonitor.yaml

```diff
@@ -0,0 +1,32 @@
+{{- with .Values.serviceMonitor }}
+{{- if .enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: mapper
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- with .Values.global.commonLabels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+    app.kubernetes.io/version: {{ .Chart.Version }}
+    app: otterize-network-mapper
+    release: prometheus
+  annotations:
+    {{- with .Values.global.commonAnnotations }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+    app.kubernetes.io/version: {{ .Chart.Version }}
+spec:
+  endpoints:
+  - port: http
```

Comment:
Same comment here re the metrics port

Reply:
This is also related to service port labelling which we are referring to in the service monitor, the otterize-network-mapper has 

```
ports:
  - name: http
    port: 9090
    protocol: TCP
    targetPort: 9090
```